<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WavegenProgressPlotter</class>
 <widget class="QWidget" name="WavegenProgressPlotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QCheckBox" name="nInsertions">
     <property name="text">
      <string>Insertions</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="nElites">
     <property name="text">
      <string>Archive size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="nReplacements">
     <property name="text">
      <string>Replacements</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="nCandidates">
     <property name="text">
      <string>Candidates</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QCustomPlot" name="plot" native="true"/>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QComboBox" name="archive"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
